functions/test/utils/AssertHelper.test.ts

```diff
@@ -0,0 +1,36 @@
+import { notNull } from "../../src/utils/AssertHelper";
+import { expect } from 'chai';
+
+describe('notNull tests', () => {
+    it('notNull(null)', () => {
```

Comment:
I would recommend that we follow some semantics for the unit tests we add. There are normally two options: `it` or `test` - that do the same thing.

`it` should preferably be followed by "should"
`test` should prefereably be followed by "given/when/then"

https://markus.oberlehner.net/blog/naming-your-unit-tests-it-should-vs-given-when-then/